### ISSUE TITLE
Build 1367 Weapon date restriction messages

### ISSUE THREAD

**Issue by SnappyComebacks** [dcs-liberation/dcs_liberation#984]
Before filing, please search the issue tracker to see if the issue has already been reported.

**Describe the bug**
A number of "this is missing a date, assuming always available" for different weapons.

**To Reproduce**
1. Create Desert War Campaign  April 18 2010
2. Enable Weapon date restricitions
3. Buy planes, etc
4. Start mission, examine logs

**Expected behavior**
Weapons to be restricted to appropriate dates.

**Additional information**

Logs:  

[missisng_dates.txt](https://github.com/Khopa/dcs_liberation/files/6332139/missisng_dates.txt)

**Version information (please complete the following information):**
 - DCS Liberation Build 1367

**Comment by ArcKnight** [comment]
Is there any issue with pulling in every weapon from pydcs and giving it a placeholder date (1930) if one doesn't already exist? I can submit a PR for this shortly, and then future requests can be more targeted towards specific weapons that are known to be out of era. Researching the in-service date of every piece of equipment modeled in DCS isn't realistic for any single person in a reasonable timeframe IMO, and this temporary solution prevents spammy log output that isn't necessarily useful.

**Comment by DanAlbert** [comment]
> Is there any issue with pulling in every weapon from pydcs and giving it a placeholder date (1930) if one doesn't already exist?

Yes, we need to be able to figure out what weapons we don't have real data for. I'd rather just remove the log.

**Comment by ArcKnight** [comment]
I'm unclear on what the functional difference between adding an obvious, declared placeholder date and removing the log statement entirely accomplishes. To be clear, I'm not talking about automatically adding things from pydcs - I'm talking about manually adding in the ~700 missing entries below the existing, organized entries in weapons.py so it's obvious they don't have real data yet. 

This also makes it easier to see how many are left without data as progress is made on research, since nobody has to scroll through hundreds of duplicates to find things without data.

If you'd rather remove the log statement, that's easy enough.

**Comment by DanAlbert** [comment]
We have a debug option that can tell us what data is missing, so the log isn't accomplishing much given how noisy it is.

Or alternatively if you want to keep the log for the player's benefit, rather than a log entry per weapon per pylon, group them up and log once? Could log only the first few to avoid logging a huge amount.

**Comment by zhexu14** [comment]
Closing as #3580 has updated all weapons data for payloads defined in resources/customized_payloads and #3578 has introduced a way to systematically look for weapons data issues.
